syntax = "proto3";

package User;

service ReqInfo {
	rpc GetInfo(C2STestInfoReq) returns(S2CTestInfoRes);
}

message C2STestInfoReq {
	uint32 ReqNum = 1;
}

message S2CTestInfoRes {
	string ResStr = 1;
}

message PacketType {
	oneof Type_oneof {
		C2SPCLoginUserReq C2SLoginUserReq = 1;
		S2CPCLoginUserRes S2CLoginUserRes = 2;
	}
}

message C2SPCLoginUserReq {
	uint32 UserID	= 1;
}

message S2CPCLoginUserRes {
	uint32  UserID		= 1;
	string  UserName	= 2;
	uint32  UserLevel	= 3;
}